mk/linux.mk

```diff
@@ -75,7 +75,7 @@ else
 RAPIDJSONCPPFLAGS=-DRAPIDJSON_HAS_STDSTRING -DRAPIDJSON_SSE42
 endif
 EIGENINCLUDES=-isystem $(CPP_SRC_HOME)/3rd_party/eigen
-EIGENCPPFLAGS=-DEIGEN_MPL2_ONLY
+EIGENCPPFLAGS=-DEIGEN_MPL2_ONLY -DEIGEN_MAX_ALIGN_BYTES=32
```

Comment:
Please merge in latest master and either add this to `mk/linux_cross_compile_linux.mk` too or else add a comment saying it's a deliberate difference from the flags used on x86_64.

Reply:
Done in [c405438](https://github.com/elastic/ml-cpp/pull/1142/commits/c405438d6cdcb4eaa49eec80eebd7e85a3881a8a).